docs/installation_guide.md

```diff
@@ -63,27 +63,31 @@ Configuration parameters are located in the file **harbor.cfg**.
-* **db_password**: The root password for the MySQL database used for **db_auth**. _Change this password for any production use!_ 
-* **max_job_workers**: (default value is **3**) The maximum number of replication workers in job service. For each image replication job, a worker synchronizes all tags of a repository to the remote destination. Increasing this number allows more concurrent replication jobs in the system. However, since each worker consumes a certain amount of network/CPU/IO resources, please carefully pick the value of this attribute based on the hardware resource of the host. 
+* **hostname**: The target host's hostname, which is used to access the Portal and the registry service. It should be the IP address or the fully qualified domain name (FQDN) of your target machine, e.g., `192.168.1.10` or `reg.yourdomain.com`. _Do NOT use `localhost` or `127.0.0.1` for the hostname - the registry service needs to be accessible by external clients!_ 
+* **ui_url_protocol**: (**http** or **https**.  Default is **http**) The protocol used to access the Portal and the token/notification service.  If Notary is enabled, this parameter has to be _https_.  By default, this is _http_. To set up the https protocol, refer to **[Configuring Harbor with HTTPS Access](configure_https.md)**.  
+* **db_password**: The root password for the PostgreSQL database used for **db_auth**. _Change this password for any production use!_ 
+* **max_job_workers**: (default value is **10**) The maximum number of replication workers in job service. For each image replication job, a worker synchronizes all tags of a repository to the remote destination. Increasing this number allows more concurrent replication jobs in the system. However, since each worker consumes a certain amount of network/CPU/IO resources, please carefully pick the value of this attribute based on the hardware resource of the host. 
 * **customize_crt**: (**on** or **off**.  Default is **on**) When this attribute is **on**, the prepare script creates private key and root certificate for the generation/verification of the registry's token. Set this attribute to **off** when the key and root certificate are supplied by external sources. Refer to [Customize Key and Certificate of Harbor Token Service](customize_token_service.md) for more info.
-* **ssl_cert**: The path of SSL certificate, it's applied only when the protocol is set to https
-* **ssl_cert_key**: The path of SSL key, it's applied only when the protocol is set to https 
+* **ssl_cert**: The path of SSL certificate, it's applied only when the protocol is set to https.
+* **ssl_cert_key**: The path of SSL key, it's applied only when the protocol is set to https.
 * **secretkey_path**: The path of key for encrypt or decrypt the password of a remote registry in a replication policy.
+* **admiral_url**: Admiral's url, comment this attribute, or set its value to NA when Harbor is standalone.
```

Comment:
We don't have do document this because it's for integration with VIC only, and will confuse the community users.

Reply:
remove from the guide